packages/gestalt/src/Typeahead.js

```diff
@@ -327,6 +330,8 @@ TypeaheadWithForwardRef.propTypes = {
   size: PropTypes.oneOf(['md', 'lg']),
   tags: PropTypes.arrayOf(PropTypes.node),
   value: PropTypes.string,
+  // eslint-disable-next-line react/forbid-prop-types
+  zIndex: PropTypes.any,
```

Comment:
Hm…we don't have a better PropType for this? Perhaps we could add something like
```js
export const UnsafeZIndexPropType = PropTypes.any;
```
in `/src/zIndex.js`, then import that where needed?

Reply:
Not at the moment, all proptypes are .any, we can discuss this. out-of-scope for this PR.